content/enterprise_influxdb/v1.9/administration/config-data-nodes.md

```diff
@@ -228,6 +228,17 @@ InfluxData recommends values ranging from `0ms` to `100ms` for non-SSD disks.
 
 Environment variable: `INFLUXDB_DATA_WAL_FSYNC_DELAY`
 
+#### `ingress-metric-by-measurement-enabled = false`
+
+When `true`, collect statistics of points written per-measurement.
+These can be accessed via the monitoring subsystem.
```

Comment:
Is the "monitoring subsystem" on of the built-in buckets? Or is this something else? Would love to link the reader somewhere useful here...

Reply:
This means you can see the metrics in `debug/vars`, and also in the `_internal` db if that is configured.